server/graphql/v2/object/Expense.ts

```diff
@@ -188,6 +188,25 @@ const GraphQLExpense = new GraphQLObjectType<ExpenseModel, express.Request>({
         type: new GraphQLNonNull(GraphQLExpenseStatus),
         description: 'The state of the expense (pending, approved, paid, rejected...etc)',
       },
+      approvedBy: {
+        type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLAccount))),
+        description: 'The accounts who approved this expense',
+        async resolve(expense, _, req) {
+          const activities: Activity[] = await req.loaders.Expense.activities.load(expense.id);
+          const approvalActivitiesSinceLastUnapprovedState = takeRightWhile(
+            activities,
+            a => a.type !== ActivityTypes.COLLECTIVE_EXPENSE_UNAPPROVED,
+          ).filter(a => a.type === ActivityTypes.COLLECTIVE_EXPENSE_APPROVED);
+
+          const approvingUserIds = uniq(approvalActivitiesSinceLastUnapprovedState.map(a => a.UserId));
```

Comment:
Pretty cool that, out of the box, this will return users in the order of approval